packages/hdwallet-native/src/native.test.ts

```diff
@@ -1,10 +1,11 @@
-import { NativeHDWallet } from "./native";
+import * as NativeHDWallet from "./native";
```

Comment:
Using `.create()` instead of the constructor 1) lets us make this import change, which will make things prettier later, and 2) tests a slightly-longer code path.